tbtadm/controller.cpp

```diff
@@ -579,6 +602,9 @@ void tbtadm::Controller::approveAll()
 
 void tbtadm::Controller::approveAll(const fs::path& dir)
 {
+    if (!sysfsDeviceExists())
+        return;
```

Comment:
approveAll() should call approveAll(const fs::path& dir), so this is not needed here